README.rst

```diff
@@ -159,6 +159,19 @@ the available settings are outlined below.
      :config
      (add-to-list 'direnv-non-file-modes 'foobar-mode))
 
+* ``warning-suppress-types``
+
+  this is an emacs variable with a list of warning types
+  which are not displayed immediately.
+  use it to hide error messages from ``direnv``.
+  example usage (with ``use-package``):
+
+  .. code-block:: elisp
+
+    (use-package direnv
+     :custom
+     (warning-suppress-types '((direnv))))
```

Comment:
this will overwrite it completely, perhaps `:config` with `(add-to-list ...)` is a better suggestion?

Reply:
OK fixed.